### ISSUE TITLE
State machine-backed stores

### ISSUE THREAD

**Issue by jmakeig** [svelte-society/recipes-mvp#18]
Example https://svelte.dev/repl/a3552021701e4b3f919b8523af618900?version=3.18.2

**Comment by swyxio** [comment]
sweet, feel free to PR :) btw you can probably import xstate in the repl and it Just Works™

**Comment by jmakeig** [comment]
> btw you can probably import xstate in the repl and it Just Works™

Yes, you should be able to, but last I tried [there was a bug (inconsistency?) in XState](https://github.com/davidkpiano/xstate/issues/787#issuecomment-553599685) that required you to package it up yourself for the Svelte REPL. (This might be fixed. I haven’t tried a more recent version.)
